vlm/src/main/scala/geotrellis/contrib/vlm/RasterSourceRDD.scala

```diff
@@ -114,6 +115,28 @@ object RasterSourceRDD {
     ContextRDD(result, layerMetadata)
   }
 
+  def tiledLayerRDD(
+    sources: RDD[RasterSource],
+    layout: LayoutDefinition,
+    resampleMethod: ResampleMethod = NearestNeighbor
+  )(implicit sc: SparkContext): MultibandTileLayerRDD[SpatialKey] = {
+    val summary = RasterSummary.fromRDD(sources)
+    val layerMetadata = summary.toTileLayerMetadata(layout, 0)._1
+
+    val tiledLayoutSourceRDD =
+      sources.map { _.tileToLayout(layout, resampleMethod) }
+
+    val rasterRegionRDD: RDD[(SpatialKey, RasterRegion)] =
+      tiledLayoutSourceRDD.flatMap { _.keyedRasterRegions() }
+
+    val tiledRDD: RDD[(SpatialKey, MultibandTile)] =
+      rasterRegionRDD
+        .groupByKey(SpatialPartitioner(summary.estimatePartitionsNumber))
```

Comment:
I would suggest not hard-coding the partitioner but having a param `partitioner: Option[Partitioner] = None` that can pick a default result. if it's set to None.